src/topTripCardList/TopTripCardList.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { cloneElement, createRef } from 'react'
```

Comment:
You don't need the `, { cloneElement, createRef }` part.

Reply:
Removed